crates/events/src/lib.rs

```diff
@@ -15,19 +15,34 @@ const EVENT_CHANNEL_CAPACITY: usize = 1024;
 #[serde(tag = "type")]
 #[serde(rename_all = "snake_case")]
 pub enum PrismEvent {
+    /// Sent when ready to sync.
     Ready,
-    BackwardsSyncStarted { height: u64 },
-    BackwardsSyncCompleted { height: Option<u64> },
+    /// Sent when backwards sync starts at the given height.
+    HistoricalSyncStarted { height: u64 },
+    /// Sent when the backwards sync completes. Is None when the sync did not find any
+    /// [`FinalizedEpochs`].
+    HistoricalSyncCompleted { height: Option<u64> },
+    /// Sent when the DA height is updated to the given height.
     UpdateDAHeight { height: u64 },
+    /// Sent when Epoch Verification starts at the given height.
     EpochVerificationStarted { height: u64 },
+    /// Sent when the Epoch Verification was successfully verified at given height.
     EpochVerified { height: u64 },
+    /// Sent when the Epoch Verification failed. Gives the height it failed at and the error.
     EpochVerificationFailed { height: u64, error: String },
```

Comment:
Can you also add the EpochVerified and EpochVerificationFailed events to the fullnode? Think it may be in the prover engine, otherwise it would also be in the syncer.